test/config_tests.ml

```diff
@@ -381,6 +381,7 @@ let key_direction () =
     Fmt.str
       {|%a
 key-direction 1
+tls-auth [inline]
```

Comment:
Did you mean to commit this line? As I understand your previous comment this makes the test fail.

Reply:
Ah yes, I did the test locally and forget to remove it, sorry.